cli/azd/internal/azdgrpc/project_service.go

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azdgrpc
```

Comment:
How do you feel about this package name?  

I wonder is something like `grpcserver` might describe what this package is for.  I think that adding the prefix `azd` to an internal azd package seems redundant.